contracts/interfaces/IArbitrumL1Bridge.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity >=0.7.0;
+pragma solidity >=0.8.0;
```

Comment:
Minor: Do we need arbitrum interface in v6?

Reply:
Just keep the interfaces for convenience. I'm ok to remove them for now util we need them again 😆